Lib/test/test_pathlib.py

```diff
@@ -746,7 +756,9 @@ class PureWindowsPathTest(_BasePurePathTest, unittest.TestCase):
 
     equivalences = _BasePurePathTest.equivalences.copy()
     equivalences.update({
-        'c:a': [ ('c:', 'a'), ('c:', 'a/'), ('/', 'c:', 'a') ],
+        './a:b': [ ('./a:b',) ],
+        'a:b:c': [ ('./b:c', 'a:'), ('b:', 'a:b:c') ],
```

Comment:
Similar to the above, I don't see why `('./b:c', 'a:')` shouldn't resolve to `a:`.